### PR TITLE
Add multiline attribute to support line breaks

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Added `multiline` attribute to support line breaks.
It uses `white-space: pre-wrap;`to preserve line breaks and spaces entered by the users in the UI.
Also updated e2e tests and screenshot to include a multiline example.

## Motivation and Context
Apps UX Improvement epic.

## How Has This Been Tested?
Locally and on stage-1 .
To validate, use Charles Proxy to map https://widgets.risevision.com/beta/components/rise-text/3/rise-text.js to https://widgets.risevision.com/staging/components/rise-text/2020.03.11.17.22/rise-text.js and check 'Main Message - Multiline' component on [stage-1 with sample template](https://apps-stage-1.risevision.com/templates/edit/25c99d9e-dc5b-47c1-992b-d37b622d5401/?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443).

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu Please review. @stulees fyi. Thanks

File: src/rise-text.js [hunk 1]
```diff
@@ -34,6 +39,9 @@ export default class RiseText extends RiseElement {
         type: Number,
         value: MAX_TEXT_SIZE,
         observer: "_checkFontSize"
+      },
+      multiline: {
+        type: Boolean
```
Comment (alex-deaconu) [inline src/rise-text.js]: @ezequielc do you think we should set a default value (`false`) for clarity? Thanks.

Comment (ezequielc) [reply src/rise-text.js]: @alex-deaconu Good suggestion. I will add that. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu Re updating readme, we have another card for documentation so I am going to add it as a new PR. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc sounds good re: readme. Thanks.

**Pull request merged**